app/views/simple_smart_answers/_current_question.html.erb

```diff
@@ -8,7 +8,7 @@
     end %>
 
   <% description = render "govuk_publishing_components/components/govspeak", {
-    content: question.body.html_safe
+    content: question.body.html_safe,
```

Comment:
not sure what's going on here? no do block and still using content param

Reply:
Good catch - thank you.